recipes/dicom2nifti/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dicom2nifti" %}
+{% set version = "2.2.1" %}
+{% set sha256 = "b712e5ee2c367d12cd0a3139bcd604daf1da454bcc8da8b204882696c8c39b77" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
`fn` is an old syntax that is no longer used, so please remove that line.

Reply:
Thanks for the review. The line was removed.